spec/views/account/account_spec.rb

```diff
@@ -49,16 +49,313 @@
       expect(page).to have_content 'Renew?'
     end
 
+    it "Displays an unchecked option to renew item request" do
+      expect(page.has_no_checked_field?('renew_items[]')).to be_truthy
+    end
+
+    it "Displays charged items as renewable" do
+      expect(page).to have_css('#item-renew .account--charged_items input')
+      expect(page.first('.account--charged_items input').value).to eq('17365')
+    end
+
     it "Displays outstanding fines" do
-      expect(page).to have_content 'No Outstanding Fines or Fees'
+      expect(page).to have_content I18n.t('blacklight.account.no_fines_fees')
     end
 
     it "Display active requests" do
       expect(page).to have_content 'Pickup Location'
     end
 
     it "Displays items for pickup" do
-      expect(page).to have_content 'No items available for pickup'
+      expect(page).to have_content 'Items Available for Pickup'
```

Comment:
I think this is true even if no items are available. Line 71 has us covered though. Can you add a spec for that tests whether `I18n.t('blacklight.account.no_pickup_items')` gets rendered when no items are available for pickup?